go/ct/st/call_journal.go

```diff
@@ -25,8 +25,9 @@ import (
 // of calls as well as the effect of future calls to be triggered
 // by CREATE and CALL expressions.
 type CallJournal struct {
-	Past   []PastCall
-	Future []FutureCall
+	Past        []PastCall
+	Future      []FutureCall
+	ToBeCreated vm.Address
```

Comment:
The "ToBeCreated" address should be a field of the "FutureCall" struct, ideally renamed to some past-time form, like `CreatedAccount`.